tests/test_space_charge_kick.py

```diff
@@ -200,15 +201,42 @@ def test_incoming_beam_not_modified():
 
 def test_gradient():
     """
-    Tests that the gradient of the track method is computed withouth throwing an error.
+    Tests that the gradient of the track method is computed accurately.
     """
-    incoming_beam = cheetah.ParticleBeam.from_parameters(
-        num_particles=torch.tensor(10_000),
-        sigma_px=torch.tensor(2e-7),
-        sigma_py=torch.tensor(2e-7),
+
```

Comment:
This is reusing the same code as in the test `test_cold_uniform_beam_expansion` above.